.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -7,18 +7,21 @@ assignees: ''
 
 ---
 
```

Comment:
> Add template for oem releases. Both in the same template, we could keep the required one and remove the other one when release issue is open

We could add a comment here to not forget this if we decide to keep both here.
```suggestion
<!--
Another release for the ownCloud Android client!

For Open releases, keep the Open Release template and remove the OEM Release one

For OEM releases, keep the OEM Release template and remove the Open Release one
-->
```

Reply:
sure, i will